### ISSUE TITLE
Navigation Service is null in App when linking to 16299+

### ISSUE THREAD

**Issue by antlhuede** [Windows-XAML/Template10#1650]
TLDR;
Reproducible bug: Navigation Service is uninitialized in View Models and Application object
Steps to Reproduce: 
1. Create UWP Template10 application
2. Set Min Version to 16299
3. Try to navigate (service will be null)

---------------------------------------------------------------------------------------------------------

Hello,

I am currently using Template10, but will be switching to Windows Template Studio and my own Navigation Solution.

This is because Template10 has been broken for builds beyond 16299 for some time now.
I am moving to strictly target .Net Standard 2.0 and my minimum version therefore must be 16299.

Unfortunately your navigation services are broken with this build. Windows Template Studio offers a much more stable and useful set of tools / application environment to work in, the only downside is the navigation services are terrible.

Given that your navigation services are good enough to be usable in production I have been utilizing them, but now that they no longer work I will be implementing my own and ditching Template10 entirely.

Just giving you a heads up, this is why I am bailing from your product.
To everyone on the Template10 Team, Thanks for all your hard work and development to date! 
It has really helped me launch my software and it really did help in the early days of development, but unfortunately I can no longer stick with using your packages at this point.

Sincerely,
Ant

**Comment by Quico20** [comment]
mmmm... I'm using Template 10.1 beyond 16299 and NavigationService has given me any trouble.
If you use T10.1 Nuget Package I'm agree you get tons of errors but surprisingly in my case it build in spite of the errors.

And I was able to test and debug my app, the thing that I did to get rid of those errors was to download T10.1 Source code and modify the code enough to get rid of all errors, and now I'm using it without any errors, Navigation Services I repeat has given me any trouble at all!!!

**Comment by panoukos41** [comment]
Hey do you create the navigation service yourself?
I remember if i didn't create it with my shell when i initialized the app i would always get that the service was null when i tried to call it.

[Try using the NavigationServiceFactory of the BootStrapper.](https://github.com/Windows-XAML/Template10/wiki/Bootstrapper#root-frame)

**Comment by Quico20** [comment]
@antlhuede man I added a Link to my template10 Custom version workable with the build 16299 or higher.
Tested by me.
Just uploaded the files you will have to set your project manually, I'm a beginner using GitHub. find instruction at #1644 
[Template10 V1.1.13-Custom](https://github.com/Quico20/Xaml-Template10.1-V1.1.13-Custom)

**Comment by antlhuede** [comment]
@panoukos41 i just wrote this bug and bailed, didnt check back or get any notifications. as an update, yes I do create the navigation service myself, it still wasn't being populated.
@Quico20 I will switch libraries before going in and editing the source as you have described. No offense, but I cannot afford to be debugging problems that are introduced in that fashion (not that you have created any, but I will not put myself in that position).
